mappings/net/minecraft/block/FireBlock.mapping

```diff
@@ -1,17 +1,26 @@
 CLASS net/minecraft/unmapped/C_dwizkkyd net/minecraft/block/FireBlock
 	FIELD f_celowrja DIRECTION_PROPERTIES Ljava/util/Map;
+	FIELD f_emnlawrv FLAME_INSTANT I
 	FIELD f_endlwine NORTH_SHAPE Lnet/minecraft/unmapped/C_zscvhwbd;
+	FIELD f_eqafjjjm BURN_INSTANT I
 	FIELD f_gqdwyhky SOUTH Lnet/minecraft/unmapped/C_xhwijdsd;
 	FIELD f_iaontxon EAST Lnet/minecraft/unmapped/C_xhwijdsd;
+	FIELD f_ikqktcwy FLAME_EASY I
 	FIELD f_isgnsdwi EAST_SHAPE Lnet/minecraft/unmapped/C_zscvhwbd;
+	FIELD f_jtmytify BURN_HARD I
```

Comment:
Feels weird, burn hard?

Reply:
Nvm it means burn hard for hard mode right

Reply:
Ye